app/uk/gov/hmrc/personaldetailsvalidation/endpoints/PersonalDetailsCollectionController.scala

```diff
@@ -27,11 +27,11 @@ class PersonalDetailsCollectionController @Inject()(page: PersonalDetailsPage,
                                                     personalDetailsSubmission: FuturedPersonalDetailsSubmission)
   extends FrontendController {
 
-  def showPage(implicit completionUrl: CompletionUrl): Action[AnyContent] = Action { implicit request =>
-    Ok(page.render)
+  def showPage(implicit completionUrl: CompletionUrl, alternativeVersion: Boolean = false): Action[AnyContent] = Action { implicit request =>
```

Comment:
`alternativeVersion` is amibguous in the sense that by looking at it we can't make out whether the default form will have nino or postcode?
Can we not call the queryParam as `withPostcode` or `postcodeVersion` so that the meaning of query param is  obvious?